pkg/apis/alerting_notifications/v0alpha1/templategroup_spec.go

```diff
@@ -0,0 +1,7 @@
+package v0alpha1
+
+// TemplateGroupSpec defines model for TemplateGroupSpec.
```

Comment:
it's not mandatory because it's set on the package level 
https://github.com/grafana/grafana/blob/b07592620279f16b0353444e7aba3c457c50d7ec/pkg/apis/alerting_notifications/v0alpha1/doc.go#L2

Reply:
add it just for uniformity.